TestVectors/dafny/TestVectors/src/WriteEsdkJsonManifests.dfy

```diff
@@ -37,7 +37,7 @@ module {:options "-functionSyntax:4"} WriteEsdkJsonManifests {
   function EncryptionContextToJson(key: string, m: mplTypes.EncryptionContext)
     : Result<seq<(string, JSON)>, string>
   {
-    var keys := SortedSets.ComputeSetToSequence(m.Keys);
+    var keys := SortedSets.ComputeSetToOrderedSequence2(m.Keys, (a, b) => a < b);
```

Comment:
Why this change?

Reply:
Recently `ComputeSetToSequence` is made method in MPL because the results are not deterministic and in Dafny we cannot call a method from a function. Our MPL submodule contains this change.

Reply:
Also, on recent commit to merge Rust work to main, this function was updated in mainline: https://github.com/aws/aws-encryption-sdk-dafny/blob/mainline/TestVectors/dafny/TestVectors/src/WriteEsdkJsonManifests.dfy#L40